port/unix/auxv.c

```diff
@@ -31,7 +31,9 @@
 #include <elf.h>
 #include <link.h>
 #include <pthread.h>
+#if defined(LINUXPPC)
```

Comment:
These changes seem to be enabling the compilation of this file on Linux 390.  How come we had to add new includes if we are compiling on Linux PPC?

Reply:
the include was there before and I assume compiles fine on power, but for some reason that header wasn't found when compiling on s390 Linux. It didn't seem to be necessary for query_auxv(AT_HWCAP). @ymanton Younes, do you know <asm/elf.h> is used in the lib on Power?

Reply:
It's not necessary on ppc64le, probably not necessary on ppc64 or ppc32 either.

Reply:
> It's not necessary on ppc64le, probably not necessary on ppc64 or ppc32 either.

@ebadyano we should probably just remove the `#if defined` and the `#include` as well then?

Reply:
@ymanton Younce, is that okay with you, if we remove that include?

Reply:
Yes it's fine.